hxinsight-extension/src/main/java/org/alfresco/hxi_connector/hxi_extension/service/config/HxInsightClientConfig.java

```diff
@@ -42,5 +43,6 @@ public HxInsightClientConfig(@NotBlank String baseUrl)
         this.agentUrl = baseUrl + "/v1/agents";
         this.questionUrl = baseUrl + "/v1/questions";
         this.answerUrl = questionUrl + "/%s/answer";
+        this.avatarUrl = agentUrl + "/%s/avatars";
```

Comment:
Given this was previously set differently then is it actually being used? Can we just delete this entry?

Reply:
Deleted 👍